rtf

```diff
@@ -763,6 +769,43 @@ Add the test to an existing scenario:
       make_scenario_file(scenario_file, test_name)
       print("Additionnaly, a scenario has been created in " + scenario_file)
 
+  def export_technique_test(self, directive_uuid, test_name, path):
+    """ Export a technique test data, for use in a technique test, in the given path """
+
+    # Test destination directory
+    if not os.path.isdir(path):
+      print("The " + path + " path does not exist.")
+      exit(1)
+
+    # Retrieve data
+    (rudder_url, token) = self.api_connection_info()
+    endpoint = RudderEndPoint(rudder_url, token, verify=False)
+
+    file_check = test_name + ".rb"
+    file_metadata = test_name + ".metadata"
+    file_directive = test_name + ".json"
+
+    # Generate metadata
+    metadata = {}
+    metadata["check"] = file_metadata
+    metadata["directive"] = file_directive
+    metadata["compliance"] = 100
+
+    make_jsonfile(os.path.join(path, file_metadata), metadata)
+
+    # Generate check file
+    make_user_testfile(os.path.join(path, file_check))
+
+    # Generate directive
+    directive = endpoint.directive_details(directive_uuid)['directives'][0]
+
+    # This is needed because of #8687, the API does not know these parameters at creation
+    del directive["isEnabled"]
+    del directive["isSystem"]
+    del directive["priority"]
+
+    make_jsonfile(os.path.join(path, file_directive), directive)
+
```

Comment:
We should print a little note here, like in export_test, to tell the user what to do next.